dev-client/src/components/links/InternalLink.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {Link} from 'native-base';
+
+type InternalLink = {
+  label: string;
+} & React.ComponentProps<typeof Link>;
+
+export default function InternalLink({label, ...props}: InternalLink) {
```

Comment:
nits for style consistency:
```suggestion
type InternalLinkProps = {
  label: string;
} & React.ComponentProps<typeof Link>;

export const InternalLink = ({label, ...props}: InternalLinkProps) => {
```

Reply:
Thanks for the catch, this is fixed now